cloudinventario

```diff
@@ -86,6 +91,27 @@ def collect(data):
      setproctitle.setproctitle(proctitle)
    return False
 
+# sentry load and apply config
+def sentryConfig(config, level):
+  dsn = (os.getenv("SENTRY_DSN") or config['sentry']['dns'])
```

Comment:
Typo: its `dsn` not `dns`

Reply:
Sorry for the mistake in spelling. In '4a7f19b' commit spelling is fixed.